src/lib/draw-and-pick.js

```diff
@@ -23,6 +23,12 @@ import {GL, glContextWithState} from 'luma.gl';
 import {getUniformsFromViewport} from './viewport-uniforms';
 import {log, getBlendMode, setBlendMode} from './utils';
 
+const MOTION_EVENTS = [
+  'dragmove',
```

Comment:
Why do we need to separate between drag and touch? These are abstract modes right?